pandora-client-web/src/graphics/room/roomCharacter.tsx

```diff
@@ -301,6 +301,16 @@ const RoomCharacterDisplay = React.forwardRef(function RoomCharacterDisplay({
 
 	const showName = useObservable(SettingDisplayCharacterName);
 
+	// NOTE: Can this be done in a cleaner way?
+	let fontScale: number = 1.0;
+	switch (interfaceChatroomCharacterNameFontSize) {
```

Comment:
I feel it might still not be big enough for some people.

Maybe we should actually increase the default a bit from the start, considering many may have smaller screens?

Then the factors could be more like: 0.6 / 1.0 / 1.4 (new default) /  1.8 / 2.2

Reply:
Committed a new version with the proposed values. 
On small screens it is still not very large.